RegExLib.v3.ncrunchsolution

```diff
@@ -0,0 +1,6 @@
+﻿<SolutionConfiguration>
+  <Settings>
+    <AllowParallelTestExecution>False</AllowParallelTestExecution>
+    <SolutionConfigured>True</SolutionConfigured>
+  </Settings>
+</SolutionConfiguration>
```

Comment:
Add *.ncrunchsolution to .gitignore?

Reply:
this added by mistake. i will add it to .gitignore

Reply:
Done [0332028](https://github.com/ardalis/RegExLib.com/pull/25/commits/0332028396fef2d2075aef0035c528ba917f6c03)